docs/versioned_docs/version-v1.3.0/02-Features/04-service-bus-extensions.md

```diff
@@ -0,0 +1,147 @@
+﻿---
+title: "Azure Service Bus Extensions"
+layout: default
+---
+
+# Azure Service Bus Extensions
+
+We provide several additional features related to message creation/sending and message/context discoverability.
+
+## Installation
+
+This features requires to install our NuGet package:
+
+```shell
+PM > Install-Package Arcus.Messaging.ServiceBus.Core
+```
+
+## Automatic tracked and correlated Service Bus messages
```

Comment:
I'd say:
Automatically tracked and correlated Service Bus messages

or 

Automatic tracking and correlating of Service Bus messages

Reply:
Also updated the EventHubs ones.

Reply:
https://github.com/arcus-azure/arcus.messaging/pull/337/commits/ed54e335aae58713ea9e0f55fd273d499b569d5d